terraform/main.tf

```diff
@@ -125,6 +125,16 @@ module "arpa_treasury_report_security_group" {
   allow_all_egress = true
 }
 
+module "digest_email_send_security_group" {
```

Comment:
Need to add this to both api and postgres, as well as digest email send security groups